scripts/install/setup.sh

```diff
@@ -1,21 +1,55 @@
 #!/usr/bin/env bash
 
-bold() {
-  echo ". $(tput bold)" "$*" "$(tput sgr0)";
-}
-
 err() {
   echo "$*" >&2;
 }
 
-~/spinnaker-for-gcp/scripts/manage/check_duplicate_dirs.sh || exit 1
-~/spinnaker-for-gcp/scripts/manage/check_git_config.sh || exit 1
+[ -n "$PARENT_DIR" ] || PARENT_DIR="$HOME"
+
+source $PARENT_DIR/spinnaker-for-gcp/scripts/manage/service_utils.sh
+
+check_for_required_binaries
+
+PARENT_DIR=$PARENT_DIR $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_git_config.sh || exit 1
 
-PROPERTIES_FILE="$HOME/spinnaker-for-gcp/scripts/install/properties"
+[ -z "$PROPERTIES_FILE" ] && PROPERTIES_FILE="$PARENT_DIR/spinnaker-for-gcp/scripts/install/properties"
 
 source "$PROPERTIES_FILE"
 
-~/spinnaker-for-gcp/scripts/manage/check_project_mismatch.sh
+check_for_shared_vpc $CI
+
+PARENT_DIR=$PARENT_DIR PROPERTIES_FILE=$PROPERTIES_FILE $PARENT_DIR/spinnaker-for-gcp/scripts/manage/check_project_mismatch.sh
+
+if [ "$CI" = true ]; then
```

Comment:
Is there a reason to only verify the operator service account has the required permissions when operating in "CI" mode?

Reply:
I think you make a good point, the operator may be a cloud shell user who is not the project owner but would need these same permissions. I will update this.